datasets/hind_encorp/README.md

```diff
@@ -0,0 +1,207 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: http://34.68.228.168:8501/
```

Comment:
Can you add the yaml tags here ?
You can use the tagger app to do so
More info about how to add the yam tages here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card